src/olympia/devhub/utils.py

```diff
@@ -170,7 +170,9 @@ def _merged_messages():
             'high': 0,
             'trivial': 0
         },
-        'detected_type': validation['metadata']['architecture'],
+        # The addons-linter only deals with WebExtensions and no longer
+        # outputs this itself, so we hardcode this.
+        'detected_type': 'webextension',
```

Comment:
It doesn't neccessarily matter but can you make this `extension` since this is the value the linter actually returned (and which is the actual type we're referring to here, see https://github.com/mozilla/amo-validator/blob/master/validator/main.py#L15 for a good mapping)